chains/evm/contracts/ccvs/OwnableRampProxy.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {RampProxy} from "./RampProxy.sol";
+
+import {Ownable2StepMsgSender} from "@chainlink/contracts/src/v0.8/shared/access/Ownable2StepMsgSender.sol";
+
+/// @notice OwnableRampProxy is a RampProxy that uses Ownable2StepMsgSender for ownership management.
+contract OwnableRampProxy is RampProxy, Ownable2StepMsgSender {
```

Comment:
Can we reduce the number of contracts and just merge this with RampProxy

Reply:
https://github.com/smartcontractkit/chainlink-ccip/pull/1182/commits/8f1762ef84e1348c20ab970fbc61f97ba2327bf7